README.md

```diff
@@ -1,20 +1,30 @@
-# torchao: PyTorch Architecture Optimization
```

Comment:
first question i always get is what does ao mean so let's answer that somewheere in the first 3 lines

Reply:
I think this got deleted by mistake, I'll add it back